tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/MessageVaultCapabilityContract.scala

```diff
@@ -0,0 +1,77 @@
+package com.linagora.tmail.james.common
+
+import io.netty.handler.codec.http.HttpHeaderNames.ACCEPT
+import io.restassured.RestAssured.{`given`, requestSpecification}
+import io.restassured.http.ContentType.JSON
+import org.apache.http.HttpStatus.SC_OK
+import org.apache.james.GuiceJamesServer
+import org.apache.james.jmap.http.UserCredential
+import org.apache.james.jmap.rfc8621.contract.Fixture.{ACCEPT_RFC8621_VERSION_HEADER, BOB, BOB_PASSWORD, DOMAIN, authScheme, baseRequestSpecBuilder}
+import org.apache.james.utils.DataProbeImpl
+import org.hamcrest.Matchers.{equalTo, hasKey}
+import org.junit.jupiter.api.{BeforeEach, Test}
+
+object MessageVaultCapabilityContract {
+
+  trait EmailRecoveryConfigured {
+
+    @BeforeEach
+    def setUp(server: GuiceJamesServer): Unit = {
+      server.getProbe(classOf[DataProbeImpl])
+        .fluent()
+        .addDomain(DOMAIN.asString())
+        .addUser(BOB.asString(), BOB_PASSWORD)
+
+      requestSpecification = baseRequestSpecBuilder(server)
+        .setAuth(authScheme(UserCredential(BOB, BOB_PASSWORD)))
+        .build
+    }
+
+    @Test
+    def shouldReturnCorrectInfoInMessageVaultCapability(): Unit = {
+      `given`()
+        .when()
+        .header(ACCEPT.toString, ACCEPT_RFC8621_VERSION_HEADER)
+        .get("/session")
+        .`then`
+        .statusCode(SC_OK)
+        .contentType(JSON)
+        .body("capabilities", hasKey("com:linagora:params:messages:vault"))
+        .body("capabilities.'com:linagora:params:messages:vault'", hasKey("maxEmailRecoveryPerRequest"))
+        .body("capabilities.'com:linagora:params:messages:vault'.maxEmailRecoveryPerRequest", equalTo(5))
+        .body("capabilities.'com:linagora:params:messages:vault'", hasKey("restorationHorizon"))
+        .body("capabilities.'com:linagora:params:messages:vault'.restorationHorizon", equalTo("PT360H"))
```

Comment:
Not OK as a duration format. 

Rather put a count of seconds? Or `15d` string?